lib/rules/no-yield-only.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGE = '{{yield}}-only templates are not allowed';
+
+module.exports = class NoMultipleEmptyLines extends Rule {
+  visitor() {
+    if (this._rawSource.trim() !== '{{yield}}') {
+      return;
```

Comment:
Ha! I didn't realize we allowed returning `undefined` here.

Reply:
hehe, yeah. was wondering if it worked and the unit tests didn't fail so I considered it okay :D